scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -30,15 +30,17 @@ sudo rm /etc/yum.repos.d/amzn2-nvidia-tmp.repo
 
 sudo yum install -y kernel-devel-$(uname -r) \
     system-release-nvidia \
+    nvidia-driver-latest-dkms \
     nvidia-fabric-manager \
     pciutils \
     xorg-x11-server-Xorg \
     docker-runtime-nvidia \
     oci-add-hooks \
     libnvidia-container \
     libnvidia-container-tools \
-    nvidia-container-runtime-hook \
-    cuda-drivers \
+    nvidia-container-runtime-hook
```

Comment:
is it okay if we add cuda packages here?

Reply:
Not super clear with it, but AL team had some issues installing it in same yum instruction. So keeping it in sync with them.